.travis.yml

```diff
@@ -32,14 +30,8 @@ node_js:
 install:
   - npm install --verbose
 
-before_script:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then npm run start-test-server; else true; fi
-
 script:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then npm test; else npm run test-open; fi
-
-after_success:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then npm run coverage; else true; fi
+  - npm test
```

Comment:
This means mac will run full test suite, which it should do. The reason we used to do it like this was because we couldn't run docker on mac for travis.